source/Octopus.Manager.Tentacle/Dialogs/ServerConnectionDialog.xaml.cs

```diff
@@ -35,7 +36,7 @@ public ServerConnectionDialog(TentacleSetupWizardModel model)
             Loaded += async (a, e) =>
             {
                 model.ContributeSensitiveValues(logger);
-                await model.VerifyCredentials(logger);
+                await model.VerifyCredentials(logger, CancellationToken.None);
```

Comment:
I am really unsure on what we can do here other than pass in `CancellationToken.None`. Is this acceptable?

Reply:
As we are no providing any way to cancel the operation, then I think this is acceptable

Reply:
yeah if we dont have an existing token handing around at this context perhaps a CancellationToken with a timeout might be a little safer.  Technically the Http requests themselves should have a timeout as well so its probably not entirely needed, but even a verification of 5 minutes would safe us from any weird hanging results.